test/Routes/ListRoutesCommandTest.php

```diff
@@ -0,0 +1,496 @@
+                false, // has-name
+                false, // has-path
+                false
+            );
+        $this->output
+            ->expects($this->atMost(2))
+            ->method('writeln');
+
+        $method = $this->reflectExecuteMethod();
+
+        self::assertSame(
+            0,
+            $method->invoke(
+                $this->command,
+                $this->input,
+                $this->output
+            )
+        );
+    }
+
+    /**
+     * @dataProvider invalidFormatDataProvider
+     * @throws ReflectionException
+     */
+    public function testThatOnlyAllowedFormatsCanBeSupplied(string $format): void
+    {
+        $routes    = [
+            new Route("/", new SimpleMiddleware(), ['GET'], 'home'),
+            new Route("/", new ExpressMiddleware(), ['GET'], 'home'),
+        ];
+        $collector = $this->createMock(RouteCollector::class);
+        $collector
+            ->expects($this->once())
+            ->method('getRoutes')
+            ->willReturn($routes);
+
+        /** @var ContainerInterface&MockObject $container */
+        $container = $this->createMock(ContainerInterface::class);
+        $container
+            ->expects($this->once())
+            ->method('get')
+            ->with(RouteCollector::class)
+            ->willReturn($collector);
+
+        /** @var ConfigLoaderInterface $configLoader */
+        $configLoader  = $this->createMock(ConfigLoaderInterface::class);
+        $this->command = new ListRoutesCommand($container, $configLoader);
+
+        $this->input
+            ->method('getOption')
+            ->willReturnOnConsecutiveCalls(
+                $format, // format
+                false, // has-middleware
+                false, // supports-method
+                false, // has-name
+                false, // has-path
+                false    // sort
+            );
+        $this->output
+            ->expects($this->once())
+            ->method('writeln')
+            ->with(
+                "Invalid output format supplied. Valid options are 'table' and 'json'"
+            );
+
+        $method = $this->reflectExecuteMethod();
+
+        self::assertSame(
+            -1,
+            $method->invoke(
+                $this->command,
+                $this->input,
+                $this->output
+            )
+        );
+    }
+
+    /**
+     * @return array<array-key,array<array-key,string>>
+     */
+    public function invalidFormatDataProvider(): array
+    {
+        return [
+            [
+                'rabbits',
+            ],
+            [
+                'tables',
+            ],
+            [
+                'toml',
+            ],
+            [
+                'yaml',
+            ],
+        ];
+    }
+
+    /**
+     * @dataProvider sortRoutingTableDataProvider
+     * @throws ReflectionException
+     */
+    public function testCanSortResults(string $sortOrder, string $expectedOutput): void
```

Comment:
```suggestion
    public function testCanSortResults(string $sortOrder): void
```
Parameter `$expectedOutput` is not used.